src/scss/cdb-utilities/helpers.scss

```diff
@@ -36,6 +37,17 @@
 .u-bSpace--m {
   margin-bottom: 8px;
 }
+.u-textSecondary {
+  color: $cTypo2;
+}
+.u-textSubtitle {
```

Comment:
As long as we don't go beyond four options, I'm ok (in that case we should rethink this kind of naming)